expressEngine/server.js

```diff
@@ -15,29 +17,30 @@ import seed from './db/seedData';
 
 // Set up the express app
 const app = express();
-
+config();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/health-check', (req, res, next) => {
+app.use('/health-check', (req, res) => {
   res.status(200).json({
     status: 'ok',
     message: 'Health is good',
```

Comment:
@aman-successive use cms for any message.

Reply:
Sir, I think this message in Server.js is used just for checking the connection and is used only once in the project since we are using cms for different module messages, it may be unnecessary to create a file for one message.

Reply:
I am approving it as I believe we are using a route helper to handle it.